### PR TITLE
Update gvs workspace description [VS-1042]

### PR DESCRIPTION
**Pull request by kbergin**
In releasing bulk ingest I noticed a few places we could add some clarification to the workspace description. This has already been updated in the workspace and can go out in github in the next release.

### CODE REVIEW

File: scripts/variantstore/beta_docs/gvs-workspace-description.md [hunk 1]
```diff
@@ -65,14 +67,16 @@ The `GvsBeta` workflow in the GVS beta workspace is pre-configured to use 10 sam
 1. Select the 'Run workflow with inputs defined by file paths' radio button.
 1. Configure the workflow inputs.
     1. Enter a **name for the callset** as a string with the format “*CALLSET_NAME*” for the `call_set_identifier` variable. This string is used as to name several variables and files and should begin with a letter. Valid characters include A-z, 0-9, “.”, “,”, “-“, and “_”.
-    1. Enter the name of your **BigQuery dataset** as a string with the format “*DATASET_NAME*” for the `dataset_name` variable.
+    1. Enter the name of your **BigQuery dataset** as a string with the format “*DATASET_NAME*” for the `dataset_name` variable. Valid characters include A-z, 0-9, “,”, “-“, and “_”.
```
Comment (mcovarr) [inline scripts/variantstore/beta_docs/gvs-workspace-description.md:70]: preexisting issue and hard to tell in GitHub, but that would-be close quote is an open quote 

```suggestion
    1. Enter the name of your **BigQuery dataset** as a string with the format “*DATASET_NAME*” for the `dataset_name` variable. Valid characters include A-z, 0-9, “,”, “-”, and “_”.
```

File: scripts/variantstore/beta_docs/gvs-workspace-description.md [hunk 2]
```diff
@@ -139,4 +143,4 @@ The workflow script is released under the Apache License, Version 2.0 (full lice
 | 09/08/2022 | Updated information on workflow outputs. | Aaron Hatcher |
 | 09/07/2022 | Added licensing information.             | Kaylee Mathews |
 | 09/06/2022 | Added note about storage cost.           | Kaylee Mathews |
-| 06/24/2022 | First release of the workspace.          | Kaylee Mathews |
\ No newline at end of file
+| 06/24/2022 | First release of the workspace.          | Kaylee Mathews |
```
Comment (mcovarr) [inline scripts/variantstore/beta_docs/gvs-workspace-description.md:146]: another preexisting issue, but this whole table could use a formatting

**Review by mcovarr — APPROVED** [review]

**Review by rsasch — APPROVED** [review]

**Review by koncheto-broad — APPROVED** [review]
LGTM

**Comment by kbergin** [comment]
Responded to remaining formatting comment. Merging.

**Pull request merged**